documentation/modules/creating-storage-mapping.adoc

```diff
@@ -6,24 +6,29 @@
 [id="creating-storage-mapping_{context}"]
 = Creating a storage mapping
 
-You can create a storage mapping by using the {ocp} web console to map source data stores to {virt} storage classes.
+You can create a storage mapping by using the {ocp} web console to map source storage domains to {virt} storage classes.
 
 .Prerequisites
 
-* Source and target providers added to the web console.
+* Source and target providers added to the {ocp} web console.
 * Local and shared persistent storage that support VM migration.
 
 .Procedure
 
-. Click *Mappings*.
-. Click the *Storage* tab and then click *Create mapping*.
-. Enter the *Name* of the storage mapping.
-. Select a *Source provider* and a *Target provider*.
-. If your source provider is VMware, select a *Source datastore* and a *Target storage class*.
+. In the {ocp} web console, click *Migration* -> *StorageMaps for virtualization*.
+. Click *Create StorageMap*.
+. Complete the following fields:
 
+* *Name*: Enter a name to display in the storage mappings list.
+* *Source provider*: Select a source provider.
+* *Target provider*: Select a target provider.
++
```

Comment:
Please answer the question below.

Reply:
@ahadas @qiyuann The reason I ask is because when i tried working with this screen I wasn't asked for the information that follows

Reply:
I see in the screenshot below that we do ask, e.g., for source storage domains for RHV - is that what you mean? we're missing OpenStack here btw. And @yaacov why is it just "source" for OpenStack?
![storage-map](https://user-images.githubusercontent.com/6020068/233279423-158969f0-7c13-4ae6-b1f3-28e0b8d556ad.png)

Reply:
@ahadas I see what happened -- the names of the lists related to source and target vary with the source provider. From the directions, i was expecting another list to open as well.

Reply:
> And @yaacov why is it just "source" for OpenStack?

looks like a bug ...
https://github.com/kubev2v/forklift-console-plugin/blob/01c39da38dea8c748b28da5e9d4bb49fbb5b8685/packages/legacy/src/common/helpers.ts#L206

Issue: https://github.com/kubev2v/forklift-console-plugin/issues/404

Reply:
I think.. but @bennyz and @mmartinv may be able to recall what was the bottom line when we talked about it few weeks ago better than me :)

Reply:
@yaacov I think   `if (sourceProviderType === 'ovirt') return `sorage domains`;`  has a typo. Should be `storage domains`, right?

Reply:
@RichardHoch thanks, I'll add the typo to the bug description